src/shared/services/configuration/endpoints.json

```diff
@@ -1,22 +1,42 @@
 {
+  "_AREAS_ENDPOINT": "Retrieves geographical areas\n\nSee https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_private_areas_geography_ for reference",
```

Comment:
Why not turn this file into a `.js` file? That way you can return an object in which you can actually place comments instead of having to resort to polluting the JSON output.

Reply:
That's better indeed!